from launch import LaunchDescription
from launch_ros.actions import Node

def generate_launch_description():
    return LaunchDescription([
        Node(
            package='ur3e_mrc',
            executable='ur3e_mrc_enme480_topics'
        )
    ])